src/phoenix/server/app.py

```diff
@@ -538,6 +561,30 @@ async def plain_text_http_exception_handler(request: Request, exc: HTTPException
     return PlainTextResponse(str(exc.detail), status_code=exc.status_code, headers=headers)
 
 
+def ensure_admin_password(db: DbSessionFactory) -> _Callback:
+    async def _() -> None:
+        assert PHOENIX_SECRET
+        loop = asyncio.get_running_loop()
+        compute = partial(compute_password_hash, password=PHOENIX_SECRET, salt=PHOENIX_SECRET)
+        hash_ = await loop.run_in_executor(None, compute)
+        password_hash = coalesce(models.User.password_hash, hash_)
+        first_admin = (
+            select(func.min(models.User.id))
+            .join(models.UserRole)
+            .where(models.UserRole.name == "ADMIN")
+            .scalar_subquery()
+        )
+        stmt = (
+            update(models.User)
+            .where(models.User.id == first_admin)
+            .values(password_hash=password_hash)
+        )
+        async with db() as session:
+            await session.execute(stmt)
+
+    return _
```

Comment:
I think closes a ticket assigned to @axiomofjoy?

Reply:
ok. fwiw, this is needed to make the integration test work